apps/microsoft-teams/frontend/src/customTypes/configPage.ts

```diff
@@ -18,6 +18,7 @@ export type SelectedEvents = {
 
 export interface TeamsChannel {
   id: string;
+  tenantId: string;
```

Comment:
Do we need to have the `tenantId` as part of the channels response? Since the response will always be scoped to the `tenantId` that's passed into the app action call as a parameter.

Reply:
We have decided to keep `tenantId` after talking with @jsdalton